### ISSUE TITLE
2 players using controller possible?

### ISSUE THREAD

**Issue by djoshdavid** [argonlefou/DemulShooter#150]
Can you use this to play 2 players for Castlevania?  how would you go about that...  thanks

**Comment by argonlefou** [comment]
https://github.com/argonlefou/DemulShooter/wiki

**Comment by djoshdavid** [comment]
so I take that as a No

**Comment by argonlefou** [comment]
Let me rephrase it :. 

> Can you use this to play 2 players for Castlevania? 

Yes

> how would you go about that... thanks

Read the Wiki

**Comment by djoshdavid** [comment]
yo I read the wiki it says press 2 player start - I am trying to use 2 xbox 360 controllers.   Only controls I see are for 1 player mouse.  Unless I am missing something.

**Comment by djoshdavid** [comment]
I go into the GUI and see a bunch of #'s no idea what I am looking at.  Which one is for Xbox 360?

**Comment by argonlefou** [comment]
Just pick a # in the list and push buttons on your xbox pad:  If you see any red button going green it's the good device. 
Use that to select the buttons you want to map as Left Mouse, etc....
Check also that the + cursor does move, you can select and change axis (left or right sticks etc ...)

**Comment by djoshdavid** [comment]
okay I got it to see the controller in the settings page for controller 2 but when I go inside the game it still in the upper left. I go back to the small window and see buttons are being pressed.  But the window is shrunk like a small cube and can't make it big or change any inputs. It looks like things are cut off in the window.  Any Ideas on how to make the window size bigger where I can change options?

**Comment by argonlefou** [comment]
You need to learn how to use DemulShooter, this is not the place to explain to every new user how to do.
There are plenty of info on the internet on top of the Wiki:  video, tutorials, discord, etc...

This place is for bug/malfunctions/features request

**Comment by djoshdavid** [comment]
Bro what I was talking about is a bug or problem with the program.  The window literally isn't big enough on the GUI and I cannot resize it.   Sorry..   if you go off on me for talking about a problem.  Thanks :)  Hopefully someone can tell me how to get around the problem.

**Comment by argonlefou** [comment]
In that case it would be better to join a screenshot because I have to admit that I don't understand at all what you are trying to explain
